staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -17,10 +17,12 @@ limitations under the License.
 package resource
 
 import (
+	"errors"
```

Comment:
This is changing `cli-runtime`, which only protects client-side for clients that use `cli-runtime`. If we don't want this to happen,  shouldn't it be done on the server?

Reply:
yup I had changes in the client and server and accidentally pushed the client ones